orttraining/orttraining/python/training/orttrainer.py

```diff
@@ -193,6 +193,9 @@ def __init__(self, model, model_desc, optim_config, loss_fn=None, options=None):
                         break
                 assert dtype is not None, f"ONNX model with unknown output type ({o_desc.name})"
 
+        from torch.utils.cpp_extension import ROCM_HOME
```

Comment:
Will this work on older versions of PyTorch that some of our users might be on? Might be worth wrapping in a `try: ... except ImportError: ....`.

Reply:
Hi Derek. Looks like it was added 2/21/2020. I wrapped it in a try-except block to make it fail-safe. Thanks.